### DISCUSSION TITLE
Are you okey?

### DISCUSSION THREAD

**Discussion by SkalskiP** [fcakyon/yolov5-pip#217] [Q&A]
@fcakyon I heard that terrible things happened in Turkey... I'm really sorry. Just wanted to ask if are you okay.

**Comment by fcakyon** [comment]
Thank you for being so supportive. Fortunately, my family and I are not affected by the earthquake. But yes, it affected ten major cities and is a sad week for everyone...

**Reply by SkalskiP** [reply]
I'm glad you're all right, and of course, I extend my condolences to your country and keep my fingers crossed for all of you.
